shared-libs/transitions/src/transitions/accept_case_reports.js

```diff
@@ -81,13 +79,21 @@ const updatePlaceUuid = (doc, registrations) => {
 };
 
 module.exports = {
+  name: 'accept_case_reports',
```

Comment:
I think we should keep the NAME constant on top of the file and just export that.  (this applies to all transitions, not just this one).

Reply:
Done!